src/main/java/com/squareup/kotlinpoet/PropertySpec.kt

```diff
@@ -125,6 +136,7 @@ class PropertySpec private constructor(builder: Builder) {
   }
 
   class Builder internal constructor(internal val name: String, internal val type: TypeName) {
+    internal var validate = true
```

Comment:
This name is too generic, I'd go for `isPrimaryConstructorParameter`, so it's clear we're dealing with a very specific type of property.